src/plonemeeting/portal/core/utils.py

```
@@ -154,27 +174,31 @@ def _get_uids_filter_url(uids):
     res = ""
     for dict_value in values:
-        res += "&{parameter}={value}".format(parameter=dict_value["parameter"], value=dict_value["value"])
+        res += "&{parameter}={value}".format(
+            parameter=dict_value["parameter"], value=dict_value["value"]
+        )
     return res
 
 
-def get_base_api_url_for_meeting_items(institution, meeting_external_uid, item_external_uids=[]):
+def get_base_api_url_for_meeting_items(
+    institution, meeting_external_uid, item_external_uids=[]
```

Ruff (0.15.14)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)